monai/transforms/spatial/dictionary.py

```diff
@@ -267,13 +269,12 @@ class Resized(MapTransform):
     Args:
         keys: keys of the corresponding items to be transformed.
             See also: :py:class:`monai.transforms.compose.MapTransform`
-        spatial_size (tuple or list): expected shape of spatial dimensions after resize operation.
-            If `spatial_size` are not defined, or smaller than 1, the transform will use the spatial size of `img`.
+        spatial_size: expected shape of spatial dimensions after resize operation.
```

Comment:
If the components of the `spatial_size` are non-positive values, the transform will use the corresponding components of `img` size. For example, `spatial_size=(32, -1)` will be adapted to `(32, 64)` if the second spatial dimension size of `img` is `64`.